src/Rpc/NodeGuardService.cs

```diff
@@ -373,6 +403,13 @@ public override async Task<OpenChannelResponse> OpenChannel(OpenChannelRequest r
                 throw new RpcException(new Status(StatusCode.Internal, "Error adding channel operation request"));
             }
 
+            if (request.Changeless)
+            {
+                // Lock the utxos
+                await _coinSelectionService.LockUTXOs(utxos, channelOperationRequest,
+                    BitcoinRequestType.ChannelOperation);
```

Comment:
cc @RodriFS

Reply:
Do you think we should start a conversation from here? When should a utxo be locked apart from those situations? @RodriFS

Reply:
I would say that we should add `PSBTSignaturesPending` and `FinalizingPSBT` to the lock. It would be a shame that you do coin selection for a multisig wallet and then someone does a withdrawal with the utxo you selected.

Reply:
New Issue created from here #275